app/controllers/search_controller.rb

```diff
@@ -125,8 +125,17 @@ def feed_content
                   # all articles, podcast episodes, and users
                   feed_content_search.concat(user_search)
                 elsif params[:class_name] == "User"
-                  # No need to check for articles or podcast episodes if we know we only want users
-                  user_search
+                  if FeatureFlag.enabled?(:search_2_users)
+                    Search::Postgres::User.search_documents(
+                      term: feed_params[:search_fields],
+                      page: feed_params[:page],
+                      per_page: feed_params[:per_page],
+                      sort_by: feed_params[:sort_by] == "published_at" ? :created_at : nil,
```

Comment:
Similar to my other comment, we can probably unify everything once the migration is complete 🎉 .